src/concrete/ml/sklearn/xgb.py

```diff
@@ -318,9 +363,17 @@ def __init__(
         self.validate_parameters = validate_parameters
         self.predictor = predictor
         self.enable_categorical = enable_categorical
-        self.use_label_encoder = use_label_encoder
```

Comment:
is xgboost not supporting this ? does xgboost never fail when given a new but unsupported attribute at instantiation ? and why did we even have that ?

Reply:
It was marked as deprecated and is now removed as far as I can see.

Reply:
but you did not update xgb in this PR 🤔  do you mean it's already deprecated in the current version we're using ?

Reply:
AH yes I got confused. So: in this PR I remove use_label_encoder from XGBRegressor because it's just not part of the xgboost.sklearn model. Must have been a copy past from the classifier.

But in the other internal PR the new xgboost version has

```python
        if use_label_encoder is True:
            raise ValueError("Label encoder was removed in 1.6.")
```

so it will disappear from both eventually.